src/net_processing.cpp

```diff
@@ -724,6 +744,41 @@ void RequestTx(CNodeState* state, const uint256& txid, int64_t nNow) EXCLUSIVE_L
     peer_download_state.m_tx_process_time.emplace(process_time, txid);
 }
 
+// Given a txid and a peer that we'd like to consider downloading a transaction
+// from: add an appropriate INV nessage to get_data if it is time to request
```

Comment:
I've got an important nessage for you: this should be "message".

Reply:
Fixed.